qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -312,13 +312,13 @@ public void allowsOnlyProperlyNamedLocalVariables() throws Exception {
                         "Name 'prolongations' must match pattern",
                         "Name 'very_long_variable_id' must match pattern",
                         "Name 'camelCase' must match pattern",
-                        "Name 'it' must match pattern '^id$|^[a-z]{3,12}$'.",
+                        "Name 'it' must match pattern",
                         "Name 'number1' must match pattern",
-                        "Name 'ex' must match pattern '^id$|^[a-z]{3,12}$'.",
-                        "Name 'a' must match pattern '^id$|^[a-z]{3,12}$'.",
-                        "Name 'ae' must match pattern '^ex|[a-z]{3,12}$'.",
-                        "Name 'e' must match pattern '^ex|[a-z]{3,12}$'.",
-                        "Name 'it' must match pattern '^id$|^[a-z]{3,}$'."
+                        "Name 'ex' must match pattern",
```

Comment:
@kitsook I see so you are ignoring the regex that comes in the error strings just for the test. Makes sense, thanks for the clarification.